### DISCUSSION TITLE
GTK-warning 이 뜹니다

### DISCUSSION THREAD

**Discussion by qwertyBBeers** [Road-Balance/gcamp_ros2_basic#26] [Q&A] [answered]
![Screenshot from 2022-07-12 14-35-07](https://user-images.githubusercontent.com/98011534/178416301-a4e82562-71b6-4a0b-a17d-253fb4816a9c.png)

다음과 같이 bash 설정 화면에 들어가지지 않습니다.

**Comment by qwertyBBeers** [comment, marked as answer]
모니터에 연결하지 않은 상태라고 하지만, 현재 노트북을 사용하고 있으며 모니터 하나 더 연결되어 있는 상태입니다

**Reply by qwertyBBeers** [reply]
display를 설정 하여야 한다고 해서 
export display=:1을 하니 해결되었습니다. 다만 왜 display=:1을 해야 해결되는 지 잘 모르겠습니다.

**Reply by kimsooyoung** [reply]
해결되셨다니 다행입니다. 해법도 올려주셔서 감사합니다.

display가 인식되는 순서가 0번 부터이어 
따라서 듀얼 모니터는 1번으로 잡히게 된 것 같습니다.
